src/test/java/com/exasol/adapter/document/documentfetcher/files/BucketfsDocumentFilesAdapterIT.java

```diff
@@ -25,7 +25,7 @@
 
 @Tag("integration")
 class BucketfsDocumentFilesAdapterIT extends AbstractDocumentFilesAdapterIT {
-    private static final String ADAPTER_JAR = "document-files-virtual-schema-dist-4.0.0-bucketfs-0.5.1.jar";
+    private static final String ADAPTER_JAR = "document-files-virtual-schema-dist-4.0.0-bucketfs-0.5.2.jar";
```

Comment:
Tip: replace hard-coded version here with https://github.com/exasol/exasol-version-number-provider.

Reply:
Hi @redcatbear:
For Version numbers of projects we need to use https://github.com/exasol/maven-project-version-getter, AFAIK
Additionally the string ADAPTER_JAR not only contains the version number of the current project but also of the document-files-virtual-schema-dist (4.0.0), which potentially can change as well. 

In summary I propose to stay with a simple string and maven goal artifact-reference-checker. 
If you agree, could you then please approve the PR?